src/App.tsx

```diff
@@ -20,20 +26,149 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+    infinite: false,
+
+  };
+
+  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const {
+      name,
+      value,
+      min,
+      max,
+    } = event.target;
+    const isValidValue = (+value >= +min && +value <= +max)
+      || name === 'infinite';
+
+    if (isValidValue) {
+      switch (name) {
+        case 'itemWidth':
+          this.setState({ itemWidth: +value });
+          break;
+
+        case 'frameSize':
+          this.setState({ frameSize: +value });
+          break;
+
+        case 'step':
+          this.setState({ step: +value });
+          break;
+
+        case 'animationDuration':
+          this.setState({ animationDuration: +value });
+          break;
+
+        case 'infinite':
+          this.setState((state) => ({ infinite: !state.infinite }));
+          break;
+
+        default:
+          return 0;
+      }
+    }
+
+    return this.state;
   };
 
   render() {
-    const { images } = this.state;
+    const {
+      images,
+      step,
+      frameSize,
+      itemWidth,
+      animationDuration,
+      infinite,
+    } = this.state;
 
     return (
       <div className="App">
         {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
+        <h1
+          data-cy="title"
+          className="title"
+        >
+          Carousel with
+          {' '}
+          {images.length}
+          {' '}
+          images
```

Comment:
Use interpolation here
```suggestion
          {`Carousel with ${images.length} images`}
```